microsoft-edge/devtools-guide-chromium/accessibility/reference.md

```diff
@@ -5,11 +5,22 @@ author: MSEdgeTeam
 ms.author: msedgedevrel
 ms.topic: conceptual
 ms.prod: microsoft-edge
-ms.date: 06/07/2021
+ms.date: 12/01/2023
 ---
 # Accessibility-testing features
 
-To test your webpages for accessibility, first make a checklist of accessibility aspects to test, and then use the relevant DevTools features to check each aspect.
+This article lists the accessibility aspects to test, and the corresponding features in Microsoft DevTools.
+
+Testing the accessibility of your website is important to ensure that people with different needs can use your website. In the best scenario, you'd have access to testers with different accessibility needs, and testers using various environments. But you can already do a lot by using automated tools.
+
+For example, the [Issues tool](../issues/index.md), [Accessibility Insights](https://accessibilityinsights.io), and [Lighthouse](https://developer.chrome.com/docs/lighthouse/overview/) are tools that automatically generate accessibility reports for a webpage.  Getting an automated report from such tools is only the beginning of your accessibility-testing journey.
```

Comment:
Lighthouse: use the external link, or link to a local topic instead?   

Very short; a jump page just to populate toc > tools > Lighthouse toc bucket:
https://review.learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/lighthouse/lighthouse-tool?branch=pr-en-us-2960
Its links go to local actual articles:

* Optimize website speed using Lighthouse
   https://review.learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/speed/get-started?branch=pr-en-us-2960
* Test accessibility using Lighthouse
   https://review.learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/accessibility/lighthouse?branch=pr-en-us-2960

Reply:
solved by removing links from intro.  
todo: 
* add Lighthouse row/link
* make sure a row/link for Issues tool: __
* * make sure a row/link for A. I. tool: __